lib/mutant.rb

```diff
@@ -52,6 +53,15 @@ module Mutant
 require 'mutant/ast/named_children'
 require 'mutant/ast/node_predicates'
 require 'mutant/ast/find_metaclass_containing'
+require 'mutant/ast/regexp'
+require 'mutant/ast/regexp/transformer'
+require 'mutant/ast/regexp/transformer/direct'
+require 'mutant/ast/regexp/transformer/named_group'
+require 'mutant/ast/regexp/transformer/options_group'
+require 'mutant/ast/regexp/transformer/quantifier'
+require 'mutant/ast/regexp/transformer/recursive'
+require 'mutant/ast/regexp/transformer/root'
+require 'mutant/ast/regexp/transformer/text'
```

Comment:
I'd love if we could get this into upstream. Having an immutable AST is a much better interface than their mutable one.

Reply:
Yeah, we may want to open an issue and discuss with the maintainers. I suspect we need changes here or upstream before we can use 2.0.